lib/utils/utils.go

```diff
@@ -23,7 +23,12 @@ func MakeRequest(method, URL string, payload interface{}) (*http.Response, error
 		return nil, fmt.Errorf("cannot create HTTP request. error: %v", err)
 	}
 
+	// For Zabbix
```

Comment:
I suggest to have headers passed by the caller here.

The function signature becomes
```go
func MakeRequest(method, URL string, payload interface{}, headers map[string][]string) (*http.Response, error) 
```

and  setting headers will read like:

```go
req.Header = http.Header(headers)
```

If you find that you're setting these headers quite a number of times, you can make 2 functions that do the requests with the headers set _e.g._ `makeZabbixRequest` and `makeRundeckRequest`

Reply:
sounds good.

Reply:
Updated in new push.